.idea/Gene-analyzer.iml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should not be branch developer

Reply:
File has been deleted.
Also, Main.java has been moved to the root directory.